recipes/_configure.rb

```diff
@@ -16,6 +16,13 @@
   end
 end
 
+ruby_block 'read splunk.secret' do
```

Comment:
This should go https://github.com/cerner/cerner_splunk/blob/23a5a65b05e030f6e8d12013e492e6c0c3b60e97/recipes/_install.rb#L81

Immediately after the splunk.secret is either written or generated.